docs/src/main/tut/const.md

```diff
@@ -247,7 +247,7 @@ implicit def constApplicative[Z : Monoid]: Applicative[Const[Z, ?]] =
 
 We have our `Applicative`!
 
-Going back to `Traverse`, we fill in the first paramter of `traverse` with `fa` since that's
+Going back to `Traverse`, we fill in the first paramater of `traverse` with `fa` since that's
```

Comment:
Oops …

It’s fixed now

-- 
Antoine Comte


On 12 novembre 2015 at 14:23:50, Cody Allen (notifications@github.com) wrote:

In docs/src/main/tut/const.md:

> @@ -247,7 +247,7 @@ implicit def constApplicative[Z : Monoid]: Applicative[Const[Z, ?]] =
>   
>  We have our `Applicative`!
>   
> -Going back to `Traverse`, we fill in the first paramter of `traverse` with `fa` since that's
> +Going back to `Traverse`, we fill in the first paramater of `traverse` with `fa` since that's
Oh actually this one is still off. Should be parameter not paramater :)

—
Reply to this email directly or view it on GitHub.